src/components/repository/IntentLabelCard.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <bh-card>
+    <div class="bh-card__left__content__flex-display">
+      <strong> {{ title }}</strong>
+      <div class="bh-card__left__content__flex-display__label">
```

Comment:
Don't use classes from another components, maybe this can be fix with grid system.